builder/src/merge.rs

```diff
@@ -83,6 +83,7 @@ impl Merger {
         target: ArtifactStorage,
         chunk_dict: Option<PathBuf>,
         config_v2: Arc<ConfigV2>,
+        expected_fs_version: Option<u32>,
```

Comment:
It seems we need to also consider other options like `chunk_size`, `digester`, I think maybe it's unnecessary to check this.

Reply:
Dropped this commit temporarily